internal/cli/hermes/plugin_template/plugin.py

```
@@ -0,0 +1,157 @@
+                "pipelock-hermes-hook binary not found; "
+                "install pipelock or set PIPELOCK_HERMES_HOOK_BIN"
+            ),
+        }
+
+    config_path = os.environ.get("PIPELOCK_HERMES_HOOK_CONFIG")
+    argv = [binary]
+    if config_path:
+        argv.extend(["--config", config_path])
+
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)